Makefile

```diff
@@ -430,7 +430,7 @@ start-e2e-local:
 	ARGOCD_TLS_DATA_PATH=/tmp/argo-e2e/app/config/tls \
 	ARGOCD_GPG_DATA_PATH=/tmp/argo-e2e/app/config/gpg/source \
 	ARGOCD_GNUPGHOME=/tmp/argo-e2e/app/config/gpg/keys \
-	ARGOCD_GPG_ENABLED=true \
+	ARGOCD_GPG_ENABLED=$(ARGOCD_GPG_ENABLED) \
```

Comment:
I think it might be a good idea to document this change somewhere. I imagine cases where users do not have `ARGOCD_GPG_ENABLED` set anywhere, and the unexpected change from `true` to `false` breaks something and they don't know why.

Reply:
Default for this is true, so should not change for most/any users? Good to query this mind you as I did need to check.

`ARGOCD_GPG_ENABLED?=true`